dotnet/src/InternalUtilities/connectors/AI/FunctionCalling/FunctionCallsProcessorLoggerExtensions.cs

```diff
@@ -0,0 +1,147 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.SemanticKernel.Connectors.FunctionCalling;
+
+[ExcludeFromCodeCoverage]
+internal static partial class FunctionCallsProcessorLoggingExtensions
```

Comment:
Is the partial needed here?
```suggestion
internal static class FunctionCallsProcessorLoggingExtensions
```

Reply:
yep, that's one of the requirements for logging source code generation - https://learn.microsoft.com/en-us/dotnet/core/extensions/logger-message-generator